SONAR-netCDF4/docs/tableBeamGroup1.adoc

```diff
@@ -95,6 +111,15 @@
 |:long_name = "Time offset that is subtracted from the timestamp of each sample" | |
 |:units = "s" | |
 | | |
+|float transmit_time_delay(ping_time, tx_beam) |O |Time delay for each transmit beam between the ping time and the transmit time
+|:long_name = "Time delay for each transmit beam between the ping time and the transmit time" | |
+|:units = "s" | |
+| | |
+|float blanking_interval(ping_time, beam) |M |Time between first sample in data array and the first valid values.
```

Comment:
What defines a valid value?

Reply:
Well, this variable was added by sverrre berg for EK80/ADCP. 
For bathymetry this matches the number of samples that are not recorded in water column datagram. Those samples are discarded. This was a number of samples for me.
In HAC format this was given as a distance. To be consistant with other variables we suggest to put this as a time.

We can rephrase the parameter to this :  
_float blanking_interval(ping_time, beam) |M |Amount of time during when samples are discarded samples. The number of discarded sample is given by blanking_interval*sample_interval_